### PR TITLE
Create Documentation Generation.md

### PR DESCRIPTION
**Pull request by mail4umar**
Added a page to explain how to create documentation using sphinx

### CODE REVIEW

File: docs/Documentation Generation.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+```
+python3 replace_sphinx_dir.py
+```
+
+- Install the updated verticapy with corrected directory paths
+```
+pip install ../.
+```
+
+- CLean the build
```
Comment (oualib) [inline docs/Documentation Generation.md:58]: Clean

File: docs/Documentation Generation.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+```
+pip install -r requirements
+```
+- Additionally install pandoc
+
+```
+apt install pandoc
+```
+
+- Edit the ``replace_sphinx_dir.py`` file to change the replacement directory to the PWD (present working directory) of docs folder
```
Comment (oualib) [inline docs/Documentation Generation.md:31]: folder.

File: docs/Documentation Generation.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+register_option(
+    Option(
+        "theme",
+        "sphinx",
+        "",
+        in_validator(["light", "dark", "sphinx"]),
+    )
+)
+```
+- Open a bash terminal and cd into ``VerticaPy/docs``. Then install all requirements
```
Comment (oualib) [inline docs/Documentation Generation.md:20]: requirements.

**Review by oualib — CHANGES_REQUESTED** [review]
small correction + please add a link to this page in the README

File: docs/Documentation Generation.md (hunk 2 above)
Comment (williammatherjones) [reply docs/Documentation Generation.md:31]: This is my opinion:  There are a number of cases where comments don't end with a period.   In general, short, single sentences don't need to have them.   If a comment has more than one sentence, it should end with a period.  Everything in between is a judgement call.

**Review by williammatherjones — COMMENTED** [review]
Looks good.   I had some comments with suggested edits.

**Review by oualib — APPROVED** [review]

**Pull request merged**